index.html

```diff
@@ -433,6 +476,127 @@ <h3 id="tables">Tables</h3>
         </table>
       </div>
 
+      <table class="custom-borders c1 c2 c3 c4 c5 c6 c7 c8">
+        <caption>Example table with custom borders and aligment</caption>
+        <colgroup>
+          <col class="border-right">
+          <col class="border-right">
+          <col class="border-right">
+          <col class="border-right">
+          <col class="border-right">
+          <col class="border-right">
+          <col class="heavy-border-right">
+          <col>
+        </colgroup>
+        <tr>
+          <th scope="col" colspan="7"></th>
+          <th>DeMorgan's Law</th>
+        </tr>
+        <tr class="border-bottom">
+          <td>$P$</td>
+          <td>$Q$</td>
+          <td>$\lnot P$</td>
+          <td>$\lnot Q$</td>
+          <td>$P \land Q$</td>
+          <td>$\lnot P \lor \lnot Q$</td>
+          <td>$\lnot (P \land Q)$</td>
+          <td>$\lnot P \lor \lnot Q \iff \lnot (P \land Q)$</td>
+        </tr>
+        <tr class="border-bottom">
+          <td rowspan="2">T</td>
+          <td>T</td>
+          <td rowspan="2">F</td>
+          <td>F</td>
+          <td>T</td>
+          <td>F</td>
+          <td>F</td>
+          <td>T</td>
```

Comment:
should be `F` instead of `T`

Reply:
Sorry. I don't understand where the error is.

Reply:
line 513, DeMorgan's Law should be `F` when both `P` and `Q` are true
<img width="607" alt="image" src="https://github.com/vincentdoerig/latex-css/assets/24668338/d6afb1da-289c-48a5-91a6-3068770cb54b">

Reply:
That last column in the table is correct (i.e., all elements must be "T").

I think the confusion arises from the styles of representing the truth tables. In Mathematical Logic, we used to write the last column (the column for the Tautological formula), with all elements equal to "true" (spared no expense):
[https://en.wikipedia.org/wiki/Tautology_(logic)#Verifying_tautologies](https://en.wikipedia.org/wiki/Tautology_(logic)#Verifying_tautologies)

But it's seems more common to avoid this last column:
[https://en.wikipedia.org/wiki/De_Morgan%27s_laws#/media/File:In_Quest_of_Univeral_Logic_Morgan.png](https://en.wikipedia.org/wiki/De_Morgan%27s_laws#/media/File:In_Quest_of_Univeral_Logic_Morgan.png)

By the way, there was a typo in the name, it should say "De Morgan". I have reformatted the table, without the last column, if you prefer this way.

Reply:
Whoops I apologise, total oversight on my part. Of course you are right! For some reason I misinterpreted the implication… I do think it is clearer now:)